lib/python/qmk/path.py

```diff
@@ -33,3 +33,17 @@ def normpath(path):
         return os.path.normpath(path)
 
     return os.path.normpath(os.path.join(os.environ['ORIG_CWD'], path))
+
+
+def cpathfiles(dir_names):
```

Comment:
This is a terrible function name, I went with what came to my mind, so if anyone has a better suggestion for it, ill rename it.

Reply:
`c_sources` maybe?   
`c_source_files`?   

I'm terrible at naming things. :D   @skullydazed is better at it.